doc/api/net.markdown

```diff
@@ -51,7 +51,8 @@ var server = net.createServer((socket) => {
 });
 
 // grab a random port.
-server.listen(() => {
+server.listen((err) => {
+  if(err) throw err;
```

Comment:
I'm also +1 on putting it on the separate line. I can go ahead and do that but it means inconsistencies with other docs. A lot of other docs use `if (err) throw err;` on a single line.

Do we want  `if (err)` checks not being single line in the docs?